buildtest/cli/build.py

```diff
@@ -308,10 +308,14 @@ def discover_buildspecs_by_tags(buildspec_cache, tagnames):
     buildspecs_by_tags = {}
 
     buildspecs = []
+    multiple_taglist = []
     # query all buildspecs from BUILDSPEC_CACHE_FILE for tags keyword and
     # if it matches input_tag we add buildspec to list
-
+  
     for name in tagnames:
+        multiple_taglist.extend(name.split(","))
```

Comment:
can we avoid declaring a new variable  `multiple_taglist` instead update tagnames variable as follows

```
tagnames = [tag.strip() for tagname in tagnames for tag in tagname.split(",")]
```

Just tried this and it does work

```
>>> tagnames
['fail, pass', 'python', 'network']
>>> tagnames = [tag.strip() for tagname in tagnames for tag in tagname.split(",")]
>>> tagnames
['fail', 'pass', 'python', 'network']
```

It works in the normal usecase where no comma is found in element of list.

```
>>> tagnames = ['python','fail']
>>> tagnames = [tag.strip() for tagname in tagnames for tag in tagname.split(",")]
>>> tagnames
['python', 'fail']

```